zerver/openapi/curl_param_value_generators.py

```diff
@@ -238,7 +238,8 @@ def upload_custom_emoji() -> Dict[str, object]:
 def deactivate_user() -> Dict[str, object]:
     user_profile = do_create_user(
         email='testuser@zulip.com', password=None,
-        full_name='test_user', realm=get_realm('zulip')
+        full_name='test_user', realm=get_realm('zulip'),
```

Comment:
Since you're touching this code anyway, do you mind re-formatting so that each arg is on its own line?

Reply:
Sure.

Reply:
Should I do it in a separate commit?

Reply:
You can just clean that up as part of your bigger commit.